components/Marquee/samples/MarqueeBehaviorSample.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="MarqueeExperiment.Samples.MarqueeBehaviorSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:CommunityToolkit.WinUI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+      xmlns:local="MarqueeExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      mc:Ignorable="d">
+
+    <win:StackPanel Padding="16">
+        <TextBlock FontWeight="Bold"
+                   Text="This Marquee will loop once when hovered." />
+        <controls:Marquee x:Name="Loop1Marquee"
+                          Background="Transparent"
+                          Behavior="Ticker"
+                          Content="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+                          FontSize="18"
+                          RepeatBehavior="1x"
+                          Speed="256">
+            <interactivity:Interaction.Behaviors>
+                <interactivity:EventTriggerBehavior EventName="PointerEntered"
+                                                    SourceObject="{Binding ElementName=Loop1Marquee}">
+                    <interactivity:CallMethodAction MethodName="StartMarquee"
+                                                    TargetObject="{Binding ElementName=Loop1Marquee}" />
+                </interactivity:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </controls:Marquee>
+
+        <TextBlock Margin="0,16,0,0"
+                   FontWeight="Bold"
+                   Text="This Marquee will loop while hovered and pause when not hovered." />
```

Comment:
I'd invert this paradigm, so you see it moving when your mouse isn't over it, as an accessibility thing folks are probably going to want to have the text stop moving if they're trying to put their mouse on it.